packages/common/src/discussions/_internal/DiscussionBusinessRules.ts

```diff
@@ -58,6 +59,28 @@ export const DiscussionPermissionPolicies: IPermissionPolicy[] = [
     entityOwner: true,
     licenses: ["hub-premium"],
   },
+  {
+    permission: "hub:discussion:canEditAccess",
+    dependencies: ["hub:discussion"],
+    authenticated: true,
+    assertions: [
+      {
+        property: "context:currentUser.privileges",
+        type: "contains-some",
+        value: [
```

Comment:
I assume the permission is used to show/hide a component, which then looks at the privs to decide if "Org" and "Everyone" are enabled...

Reply:
yes, that is correct 👍